src/app/code/community/Divante/VueStorefrontIndexer/etc/config.xml

```diff
@@ -229,12 +229,12 @@
                                 <transaction_key>Divante_VueStorefrontIndexer_Model_Indexer_Datasource_Transactionkey</transaction_key>
                             </datasources>
                         </cms_block>
-                        <cms_page>
-                            <mapping>Divante_VueStorefrontIndexer_Model_Index_Mapping_Cms_Page</mapping>
+                        <review>
+                            <mapping>Divante_VueStorefrontIndexer_Model_Index_Mapping_Review</mapping>
                             <datasources>
                                 <transaction_key>Divante_VueStorefrontIndexer_Model_Indexer_Datasource_Transactionkey</transaction_key>
                             </datasources>
-                        </cms_page>
+                        </review>
                     </types>
```

Comment:
@qbo-tech Please restore changes for "cms_page" configuration, thanks!

Reply:
I made a fix, and create new PR
https://github.com/DivanteLtd/magento1-vsbridge-indexer/pull/28